### PR TITLE
docs: Updated doc with changes in TS client

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/typescript/crud.md [hunk 1]
```diff
@@ -46,21 +46,35 @@ const insertedOrReplacedDoc: User = await users.insertOrReplace({
...
 ```
 
 ### Read documents
 
-To read multiple documents, use the `read()` API.
+To read multiple documents, use the `find()` or `findStream()` API.
 
 ```typescript
-users.read(
+const result: Promise<User> = users.find({
```
Comment (ovaistariq) [inline docs/typescript/crud.md:63]: Just keep users.find call without the assignment

File: docs/typescript/crud.md [hunk 2]
```diff
@@ -46,21 +46,35 @@ const insertedOrReplacedDoc: User = await users.insertOrReplace({
...
+    },
+    {
+      name: "emma",
+    },
+  ],
+});
+```
+
+```typescript
+users.findStream(
```
Comment (ovaistariq) [inline docs/typescript/crud.md:77]: Better to have example of both findMany and findStream

File: docs/typescript/crud.md [hunk 3]
```diff
@@ -46,21 +46,35 @@ const insertedOrReplacedDoc: User = await users.insertOrReplace({
...
-const user: User = await users.readOne({
+const user: User = await users.findOne({
   userId: 1,
 });
 ```
 
 ### Read documents
 
-To read multiple documents, use the `read()` API.
+To read multiple documents, use the `find()` or `findStream()` API.
```
[the remark this answers is not in the document]
Comment (JigarJoshi) [reply docs/typescript/crud.md]: fixed.

File: docs/typescript/crud.md (hunk 2 above)
Comment (JigarJoshi) [reply docs/typescript/crud.md]: fixed.

File: docs/typescript/crud.md (hunk 1 above)
Comment (JigarJoshi) [reply docs/typescript/crud.md]: fixed.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**